validation/src/test/scala/hmda/validation/rules/lar/quality/Q068Spec.scala

```diff
@@ -0,0 +1,58 @@
+package hmda.validation.rules.lar.quality
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.LarEditCheckSpec
+
+class Q068Spec extends LarEditCheckSpec {
+  // Cases meeting preconditions
+  property("passes when applicant is not a natural person and coapplicant is a natural person") {
```

Comment:
The name doesn't quite match the code. "coapplicant is a natural person or does not exist" would match the code. Or, it might be better to change the code to match the name, since the does-not-exist case is covered in the next one. Hm.